py_ecc/optimized_bls12_381/optimized_field_elements.py

```diff
@@ -36,7 +28,7 @@ def __init__(self, n):
             self.n = n.n
         else:
             self.n = n % field_modulus
-        assert isinstance(self.n, int_types)
+        assert isinstance(self.n, int)
```

Comment:
Not sure it's worth double-checking, but thought I would add the reminder that `isinstance(bool(), int) == True`